project1-list/app/src/main/java/net/serkanbal/project1/MainActivity.java

```diff
@@ -0,0 +1,204 @@
+                if (convertView == null) {
+                    LayoutInflater inflater = LayoutInflater.from(MainActivity.this);
+                    convertView = inflater.inflate(R.layout.mainlistview_layout, null);
+                }
+
+                TextView textView = (TextView) convertView.findViewById(android.R.id.text1);
+                textView.setText(mToDoList.get(position).getToDoListName());
+
+                return convertView;
+            }
+
+        };
+
+        //Set the adapter
+        mToDoListListView = (ListView) findViewById(R.id.list_view);
+        mFab = (FloatingActionButton) findViewById(R.id.fab);
+        mTextView = (TextView) findViewById(R.id.maintext);
+        mToDoListListView.setAdapter(mToDoListAdapter);
+
+        mFab.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View v) {
+                LayoutInflater layoutInflater = LayoutInflater.from(MainActivity.this);
+                final View promptsView = layoutInflater.inflate(R.layout.dialoglist_layout, null);
+                AlertDialog.Builder alertDialogBuilder = new AlertDialog.Builder(
+                        MainActivity.this);
+
+                // set prompts.xml to alertdialog builder
+                alertDialogBuilder.setView(promptsView);
+
+                // set dialog message
+                alertDialogBuilder
+                        .setCancelable(false)
+                        .setPositiveButton("OK",
+                                new DialogInterface.OnClickListener() {
+                                    public void onClick(DialogInterface dialog,int id) {
+                                        EditText input = (EditText) promptsView.findViewById(R.id.editTextDialogUserInput);
+                                        String inputUser = input.getText().toString();
+                                        if (inputUser.isEmpty()) {
+                                            Toast.makeText(MainActivity.this,
+                                                    "List Name can't be empty!",
+                                                    Toast.LENGTH_SHORT).show();
+                                        } else {
+                                            mToDoList.add(new ToDoList(inputUser));
+                                            mTextView.setBackgroundColor(Color.parseColor("#fffafafa"));
+                                            mTextView.setText("");
+                                            mToDoListAdapter.notifyDataSetChanged();
+                                        }
+                                    }
+                                })
+                        .setNegativeButton("Cancel",
+                                new DialogInterface.OnClickListener() {
+                                    public void onClick(DialogInterface dialog,int id) {
+                                        dialog.cancel();
```

Comment:
You can actually just put `null` for the OnClickListener for the cancel button; either way the dialog will just close